### PR TITLE
adjusted the line height

### PR DESCRIPTION
**Pull request by anubhav1206**

### CODE REVIEW

**Review by colinkiama — CHANGES_REQUESTED** [review]
Sorry, my fault, I should have been more clear.

The changes you've made to these files should be reverted. Only `sass/css/_base.scss` needs to be modified.

Modify the `line-height` set on `body`.

`1` is a bit too low for `line-height`. Even Google Chrome's default `line-height` is `1.2`. Maybe try `1.3`?

**Comment by anubhav1206** [comment]
sure will make the changes

> Sorry, my fault, I should have been more clear.
> 
> The changes you've made to these files should be reverted. Only `sass/css/_base.scss` needs to be modified.
> 
> Modify the `line-height` set on `body`.
> 
> `1` is a bit too low for `line-height`. Even Google Chrome's default `line-height` is `1.2`. Maybe try `1.3`?

**Comment by anubhav1206** [comment]
done, kindly check

File: sass/css/_reset.scss [hunk 1]
```diff
@@ -22,7 +22,7 @@ Use a better box model (opinionated).
 */
 
 html {
-  line-height: 1.15; /* 1 */
+  line-height: 1; /* 1 */
```
Comment (colinkiama) [inline sass/css/_reset.scss:25]:
```suggestion
  line-height: 1.15; /* 1 */
```

File: sass/css/index.scss [hunk 2]
```diff
@@ -1,3 +1,6 @@
+*{
+  line-height: 1;
```
Comment (colinkiama) [inline sass/css/index.scss:2]:
```suggestion
```

File: sass/css/index.scss [hunk 3]
```diff
@@ -1,3 +1,6 @@
...
+}
```
Comment (colinkiama) [inline sass/css/index.scss:3]:
```suggestion
```

File: sass/css/_reset.scss (hunk 1 above)
Comment (colinkiama) [reply sass/css/_reset.scss:25]: Change this back

File: sass/css/index.scss (hunk 3 above)
Comment (colinkiama) [reply sass/css/index.scss:3]: Remove this change

**Review by colinkiama — CHANGES_REQUESTED** [review]
Almost there...

After making these final changes, your contribution will be ready to merge 😄️.

**Comment by anubhav1206** [comment]
> Almost there...
> 
> After making these final changes, your contribution will be ready to merge 😄️.

done, kindly check

File: sass/css/index.scss [hunk 4]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (colinkiama) [inline sass/css/index.scss:1]:
```suggestion
```

**Review by colinkiama — APPROVED** [review]
Perfect! Thank you!

**Pull request merged**